tests/monitoring_test.go

```diff
@@ -0,0 +1,183 @@
+package tests
+
+import (
+	"context"
+	"crypto/tls"
+	"net"
+	"net/http"
+	"time"
+
+	promApi "github.com/prometheus/client_golang/api"
+	promApiv1 "github.com/prometheus/client_golang/api/prometheus/v1"
+	promConfig "github.com/prometheus/common/config"
+	apps "k8s.io/api/apps/v1"
+	"k8s.io/apimachinery/pkg/types"
+	sspv1beta1 "kubevirt.io/ssp-operator/api/v1beta1"
+	"kubevirt.io/ssp-operator/internal/operands/metrics"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var promClient promApiv1.API
```

Comment:
Can you move this var to `test_suite_test.go`, since its value is written in that file.

Reply:
done